torchvision/models/segmentation/segmentation.py

```diff
@@ -117,8 +139,13 @@ def fcn_resnet50(pretrained=False, progress=True,
     return _load_model('fcn', 'resnet50', pretrained, progress, num_classes, aux_loss, **kwargs)
 
 
-def fcn_resnet101(pretrained=False, progress=True,
-                  num_classes=21, aux_loss=None, **kwargs):
+def fcn_resnet101(
+    pretrained: bool = False,
+    progress: bool = True,
+    num_classes: int = 21,
+    aux_loss: bool = False,
```

Comment:
Technically this is a BC-breaking change as you modify the default value of the method. Why not keep it as it was?